src/Aquifer.Public.API/Endpoints/Resources/Get/Endpoint.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Aquifer.Common.Services;
+using Aquifer.Data;
+using FastEndpoints;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Get;
+
+public class Endpoint(AquiferDbContext dbContext, IResourceContentRequestTrackingService trackingService) : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Get("/resources/{ContentId}");
+        Description(d => d.ProducesProblemFE(404));
+        Summary(s =>
+        {
+            s.Summary = "Get specific resource information.";
+            s.Description =
+                "For a given resource id, return the data for that resource. This can be text content as well as CDN links for image, audio, and video media types.";
+        });
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        Response = await ResourceHelper.GetResourceContentAsync(dbContext,
+            new ByLanguage.Request
```

Comment:
It feels weird for what is essentially the parent implementation to be calling into the helper with the "child" request class `ByLanguage.Request`. Could we add another signature in the helper?